plugins/gcloud/app/decorators/project_decorator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+Project.class_eval do
+  validate :validate_not_using_gcb_and_external
+
+  private
+
+  def validate_not_using_gcb_and_external
+    return unless build_with_gcb && docker_image_building_disabled
+    errors.add(:build_with_gcb, "cannot be enabled when building is disabled")
```

Comment:
None of the checkboxes say anything about building being disabled.

The text of the checkboxes is:
* Docker images built externally 
* Include in New Deploy Groups 
* Build docker on GCB 

The second checkbox doesn't have anything to do with the first two, which is confusing enough.

So at the very least, this error message should be "cannot be enabled when Docker images are built externally."